Packs/SimpleSFTP/Integrations/SimpleSFTP/SimpleSFTP.py

```diff
@@ -0,0 +1,43 @@
+import demistomock as demisto  # noqa: F401
+import pysftp
```

Comment:
According to [this](https://stackoverflow.com/questions/48434941/pysftp-vs-paramiko#48437583) answer it seems better to use `Paramiko`